packages/qb/pqb/src/sql/rawSql.ts

```diff
@@ -95,12 +94,21 @@ export class RawSQL<
   }
 }
 
+// Argument type for `sql` function.
+export type RawSQLArgs = TemplateLiteralArgs | SimpleRawSQLArgs;
+
+export type SimpleRawSQLArgs = [
+  { raw: string; values?: Record<string, unknown> },
+];
+
+function isSimpleRawSQLArgs(args: RawSQLArgs): args is SimpleRawSQLArgs {
+  return typeof args[0].raw === 'string';
+}
```

Comment:
Could you move these into `orchid-core`?
`orchid-core` is for all things that will be common across different dbs, for the case of our descendants will decide to support MySQL.